recipes/icu/all/conanfile.py

```diff
@@ -58,6 +58,16 @@ def _enable_icu_tools(self):
     def _with_unit_tests(self):
         return not self.conf.get("tools.build:skip_test", default=True, check_type=bool)
 
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "16",
+            "msvc": "192",
+            "gcc": "8",
+            "clang": "9",
+            "apple-clang": "12" if Version(self.version) >= "6.5.0" else "11"
```

Comment:
This just needs adapted from the Qt 6 recipe.

```suggestion
            "apple-clang": "11"
```

Reply:
Thank you so much! This makes perfect sense now that I understand what went wrong. I was having trouble understanding what the difference from Conan 1 to Conan 2 causing this was from just the logs.